README.md

```diff
@@ -9,7 +9,7 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 ![Lint Check (pre-commit)](https://github.com/bemanproject/any_view/actions/workflows/pre-commit.yml/badge.svg)
 ![Standard Target](https://github.com/bemanproject/beman/blob/main/images/badges/cpp29.svg)
 
-**Implements**: `std::ranges::any_view` proposed in [any_view (P3411R2)](https://wg21.link/p3411r2).
+**Implements**: `std::ranges::any_view` proposed in any_view (P3411).
```

Comment:
```suggestion
**Implements**: `std::ranges::any_view` proposed in any_view ([P3411](https://wg21.link/P3411)).
```

Up to you but this way the link tracks the latest version of the paper.

Reply:
I'm aware, and given that no version of the paper yet exists with this wording, I have no appropriate link to provide here.

Reply:
I see that's fair -- I don't think I'd remove the link for that though...